SpellingBee/create_spelling_data.py

```diff
@@ -0,0 +1,320 @@
+import torch
+import numpy as np
+import os
+import argparse
+import itertools
+from tqdm.auto import tqdm
+
+import nltk
+from nltk.stem import WordNetLemmatizer
+from nltk.corpus import wordnet
+
+lemmatizer = WordNetLemmatizer()
+
+
+def preprocess_dic(input_dic):
```

Comment:
Seems like this should be `input_dict_path`. Also please provide some information how input_dict should look, as I dont understand how `x.split(' ')[0]` is what you need to apply on it.